images/manageiq-base/Dockerfile

```diff
@@ -78,6 +94,7 @@ WORKDIR ${APP_ROOT}
 RUN source /etc/default/evm && \
     export RAILS_USE_MEMORY_STORE="true" && \
     gem install bundler -v ">=1.16.2" && \
+    if [ ${ARCH} = "ppc64le" ] ; then gem install sassc  -- --disable-march-tune-native ; fi && \
```

Comment:
What is this used for and why is it being installed here rather than as a part of the bundle?

Reply:
There is more context around why this was needed here https://github.com/ManageIQ/manageiq/issues/19520
Essentially the build fails while installing (or building) this gem if the --disable-march-tune-native flag is not set for ppc64le